src/openrct2/network/DiscordService.cpp

```diff
@@ -7,6 +7,7 @@
  * OpenRCT2 is licensed under the GNU General Public License version 3.
  *****************************************************************************/
 
+#include "openrct2/localisation/Formatting.h"
```

Comment:
This needs to read `#include "../localisation/Formatting.h"` and should be put around line 19, in alphabetical order.